jobs/config.json

```diff
@@ -892,6 +892,16 @@
   ]
 },
 
+"ci-kubernetes-e2e-prow-canary": {
+  "scenario": "kubernetes_prow",
```

Comment:
Prow handles them and bind them inside docker env already, the scenario does not need to worry about them anymore.